plugins/notification-resources/src/inboxNotificationsClient.ts

```diff
@@ -258,7 +258,9 @@ export class InboxNotificationsClientImpl implements InboxNotificationsClient {
     const ops = getClient().apply(generateId())
 
     try {
-      const inboxNotifications = get(this.inboxNotifications) ?? []
+      const inboxNotifications = await ops.findAll(notification.class.InboxNotification, {
```

Comment:
Let's add projections to have only _id, space, _class?

Reply:
Done.